src/chips.ts

```diff
@@ -269,18 +264,18 @@ export class Chips extends Component<ChipsOptions> {
   static _handleChipsBlur(e: Event) {
     if (!Chips._keydown && document.hidden) {
       const chips = (<HTMLElement>e.target).closest('.chips');
-      const currChips: Chips = (chips as any).M_Chips;
+      const currChips: Chips = chips['M_Chips'];
```

Comment:
I think we might need to revert this, since it introduces backwards compatibility issues with applications using the previous object orientated approach, propose to just set it to warnings until we can declare the correct type definition